pre_cleanup.lua

```diff
@@ -0,0 +1,47 @@
+local os = require('os')
+
+local function clean_state(self)
+    box.space._space:pairs():map(function(tuple)
+        return tuple[3]
```

Comment:
It is more or less clear, but I think the rule of thumb is to avoid such unnamed constants in any code (maybe except some bit manipulation involving constants 8, 16 and so on).

Reply:
My comment was about constant 3. I would define a 'constant' like `local _SPACE_NAME_FN = 3` above and use it here.